tcms/testruns/models.py

```diff
@@ -309,6 +311,10 @@ def stats_caseruns_status(self, case_run_statuses=None):
 
 
 class TestCaseRunStatus(TCMSActionModel):
+    FAILED = 'failed'
+    BLOCKED = 'blocked'
+    PASSED = 'passed'
```

Comment:
These are status names and are always upper case.

Reply:
`@classmethod def id_failed(cls): return cls._status_to_id('failed')` This is the implementation of the method I just deleted. I took the status string from there. Will change it now, but has it not been working propertly before?